act/virtualNetwork.tf

```diff
@@ -17,89 +17,90 @@ resource "azurerm_resource_group" "resource_group" {
 }
 
 resource "azurerm_virtual_network" "example" {
-  name = "vn-test"
-  address_space = ["10.0.0.0/16"]
-  location = "East US"
+  name                = "vn-test"
+  address_space       = ["10.0.0.0/16"]
+  location            = "East US"
   resource_group_name = "testrg"
 }
 
 resource "azurerm_subnet" "example" {
-  name = "subnet-test"
-  resource_group_name = "testrg"
+  name                 = "subnet-test"
+  resource_group_name  = "testrg"
   virtual_network_name = azurerm_virtual_network.example.name
-  address_prefixes = ["10.0.0.0/24"]
+  address_prefixes     = ["10.0.0.0/24"]
 }
 
 resource "azurerm_network_interface" "ni_linux" {
-  name = "linux-test"
-  location = "East US"
+  name                = "linux-test"
+  location            = "East US"
   resource_group_name = "testrg"
   ip_configuration {
-    name = "internal"
-    subnet_id = azurerm_subnet.example.id
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.example.id
     private_ip_address_allocation = "Dynamic"
   }
 }
 
 resource "azurerm_network_interface" "ni_win" {
-  name = "win-test"
-  location = "East US"
+  name                = "win-test"
+  location            = "East US"
   resource_group_name = "testrg"
   ip_configuration {
-    name = "internal"
-    subnet_id = azurerm_subnet.example.id
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.example.id
     private_ip_address_allocation = "Dynamic"
   }
 }
 
 resource azurerm_network_security_group "bad_sg" {
-  location = "East US"
-  name = "test-nsg"
+  location            = "East US"
+  name                = "test-nsg"
   resource_group_name = "testrg"
   security_rule {
-    access = "Allow"
-    direction = "Inbound"
-    name = "AllowSSH"
-    priority = 200
-    protocol = "TCP"
-    source_address_prefix = "*"
-    source_port_range = "*"
-    destination_port_range = "22-22"
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure that SSH access is restricted from the internet'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

 <!-- 3e3d5484-9b1e-48a8-8f5e-916656707c93 -->